bundle-workflow/tests/tests_system/test_execute.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import subprocess
+import unittest
+from unittest.mock import patch
+
+from system.execute import execute
+
+
+class TestExecute(unittest.TestCase):
+    def test_current_arch(self):
```

Comment:
The name of the function should be `test_execute`, because you're testing `system.execute`, not the fact that you're able to get the current arch.

Reply:
This is a miss due to copy paste from test_arch.py, will fix.